### ISSUE TITLE
Properly style Animations side panel

### ISSUE THREAD

**Issue by SebastianZ** [firebug/firebug.next#283]
The *Animations* side panel within the *Inspector* panel should look more Firebug-ish.

Looks like:
![Current display of Animations side panel](https://cloud.githubusercontent.com/assets/958943/5797432/179a672e-9fba-11e4-8485-2803b2c02982.png)
![Current display of Animations side panel when there are no animations](https://cloud.githubusercontent.com/assets/958943/5797433/1ee5e22e-9fba-11e4-82de-b4a04304456e.png)

Should look more like this:
![Proposed display of Animations side panel](https://cloud.githubusercontent.com/assets/958943/5797438/333556f6-9fba-11e4-9be1-6b4b382c7801.png)
![Proposed display of Animations side panel when there are no animations](https://cloud.githubusercontent.com/assets/958943/5797439/34b9e910-9fba-11e4-88f0-25735107a47d.png)

For reference, in the light theme it looks like this:
![Display of Animations side panel using the light theme](https://cloud.githubusercontent.com/assets/958943/5797503/7f98ff10-9fbb-11e4-87e2-761cbec7d22b.png)
![Display of Animations side panel when there are no animations using the light theme](https://cloud.githubusercontent.com/assets/958943/5797504/810ff54c-9fbb-11e4-8135-20c2f4407238.png)

**Test case:**
https://developer.mozilla.org/samples/cssref/animations/cssanim3.html

**Tasks are:**
* Change font family and color of text when there are animations
* Style stop/continue button
* Fix and style animation slider
* Change font family, size, color and adjustment of text when there are no animations
* Replace inspect button in case there are no animations by linked text

Sebastian

**Comment by fflorent** [comment]
Thanks for the report, Sebastian. I take a look at this issue.

Florent

**Comment by fflorent** [comment]
> Replace inspect button in case there are no animations by linked text

@SebastianZ What are the reasons you prefer a linked text in place of the inspect button is a good idea? 

I started to work on it, and what I have in mind is rather something like below:

![profile_picture_3](https://cloud.githubusercontent.com/assets/371705/5892231/9ee88506-a4b7-11e4-9f11-2eed65d6be3f.png)

What do you think?

Florent

**Comment by fflorent** [comment]
> Fix and style animation slider

Pull request #286 actually solves the display of the slider.

Florent

**Comment by fflorent** [comment]
Screenshots of the current prototype:

![pick another element unchecked](https://cloud.githubusercontent.com/assets/371705/5893418/621c5cac-a4e3-11e4-98a7-f122280a6d8e.png)

![pick another element checked](https://cloud.githubusercontent.com/assets/371705/5893420/75826fe8-a4e3-11e4-9d49-b1a4e509b842.png)

![animation player running](https://cloud.githubusercontent.com/assets/371705/5893421/884d804a-a4e3-11e4-9fd4-81576c7d15ed.png)

![animation player paused](https://cloud.githubusercontent.com/assets/371705/5893423/9961dab6-a4e3-11e4-985d-744f2c7ebe02.png)

Note that this work depends on #286.

Florent

**Comment by janodvarko** [comment]
Good progress!
My comments:

1. The inspector button should show a border when hovered by mouse (just like on the main toolbar)
2. The font + color for "No animations were found for the current element. Pick another element from the page." - should be the same as for e.g. "There are no event listeners directly on this element." in the Events panel in Firebug 2
3. Can we append more space (padding-top and bottom) for the labels: Animation, Duration, etc. at the top of the panel. It's a bit jammed there. Also the font for these labels should be different (you could use the same as for point 2).

Honza

**Comment by fflorent** [comment]
> 1. The inspector button should show a border when hovered by mouse (just like on the main toolbar)

Indeed. Thanks!

> 2. The font + color for "No animations were found for the current element. Pick another element from the page." - should be the same as for e.g. "There are no event listeners directly on this element." in the Events panel in Firebug 2

Sounds a good idea. 
Just the color is `#888`, which looks a bit too light and may be hard to read. How about `#555`?

> 3. Can we append more space (padding-top and bottom) for the labels: Animation, Duration, etc.

Done

> Also the font for these labels should be different (you could use the same as for point 2).

Done (with same remark as for point 2).

Florent

**Comment by aliuk2012** [comment]
Just a quick suggestion about the font colour. Maybe add it to the css variables I created (https://github.com/firebug/firebug.next/blob/master/chrome/skin/classic/shared/firebug-style-variables.css) so we can reuse the colouring and be consistent.

**Comment by fflorent** [comment]
I agree (I think I have left a comment about it). We would probably have to
know where the variable is used and evaluate the impacts of this change.

Florent

**Comment by SebastianZ** [comment]
> @SebastianZ What are the reasons you prefer a linked text in place of the inspect button is a good idea?

Sorry, I missed your comment earlier. The reason was to have a consistent UI as in other parts links are used for similar messages instead of a button.

>> 2. The font + color for "No animations were found for the current element. Pick another element from the page." - should be the same as for e.g. "There are no event listeners directly on this element." in the Events panel in Firebug 2

> Sounds a good idea. 
Just the color is `#888`, which looks a bit too light and may be hard to read. How about `#555`?

Gray color may indicate that the panel is disabled. I'd rather keep the text dark.

Sebastian

**Comment by janodvarko** [comment]
Nice, we are close, more comments:

1. The top-padding for animation-title should be 6px
2. The color for animation-title should be black
3. The color of time-display should also be black
4. The "no animation" text should be bold (just like for the Events panel for FB2)

Honza

**Comment by SebastianZ** [comment]
> 4. The "no animation" text should be bold (just like for the Events panel for FB2)

Firebug 2 is inconsistent in this regard. The *Selectors* side panel uses normal font width for example. And I believe normal font width looks a bit better. Bold is used more for emphasizing.

Sebastian

**Comment by fflorent** [comment]
> 1. The top-padding for animation-title should be 6px

Done

> 2. The color for animation-title should be black

Done

> 3. The color of time-display should also be black

Done

> 4. The "no animation" text should be bold (just like for the Events panel for FB2)

Hmm... I agree with Sebastian on this. How about `#555` (kind of dark-grey) like below?

![no animation text](https://cloud.githubusercontent.com/assets/371705/5966477/ef778738-a804-11e4-96ed-dafd8d126afb.png)

@janodvarko feel free to merge. I'd rather let you merge and modify my patch (and debating later) than leaving the display by default for the beta.

Florent

**Comment by janodvarko** [comment]
> @janodvarko feel free to merge. I'd rather let you merge and modify my patch
> (and debating later) than leaving the display by default for the beta.

I don't see your commit link, but in any case both bold/non-bold is fine for me, so you can merge into the master.

Thanks
Honza

**Comment by fflorent** [comment]
> I don't see your commit link

The commit is referenced above my previous message.

The merge is done. But I still should do some tests, I guess. I will probably have to copy the tests of the DevTools...

Florent

**Comment by janodvarko** [comment]
Looks nice, thanks!

> But I still should do some tests, I guess. 

That would be great

> I will probably have to copy the tests of the DevTools...

Please create a new issue for this. We can reuse all the API for other tests (we can also discuss on our meeting today)

I guess I can close this one now.

Honza

**Comment by janodvarko** [comment]
$fixed firebug-3.0.0-alpha.8
